docs/griptape-framework/structures/tasks.md

```diff
@@ -146,3 +146,254 @@ agent.run()
                              Output: The summary of the webpage https://www.griptape.ai has been successfully stored in a file named
                              griptape.txt.
 ```
+
+
+## Extraction Task
+
+To extract information from a text, you can use the ExtractionTask. This task allows us to pass in an Extraction Engine and a set of arguments to the engine.
+
+```python
+from griptape.tasks import ExtractionTask
+from griptape.structures import Agent
+from griptape.engines import CsvExtractionEngine, JsonExtractionEngine
+
+# Instantiate the CSV extraction engine
+csv_extraction_engine = CsvExtractionEngine()
+
+# Define some CSV data and columns
+csv_data = """
+Name, Age, Address
+John, 25, 123 Main St
+Jane, 30, 456 Elm St
+"""
+
+columns = ["Name", "Age", "Address"]
+
+
+# Create an agent and add the ExtractionTask to it
+agent = Agent()
+agent.add_task(
+    ExtractionTask(
+        input_template="{{ csv_data }}",
```

Comment:
Remove this parameter. Pass `csv_data` into `run`

Reply:
done